torch/onnx/utils.py

```diff
@@ -1015,6 +1015,15 @@ def _should_aten_fallback(ns, op_name, opset_version, operator_export_type):
     return is_onnx_aten_export or (not is_exportable_aten_op and is_aten_fallback_export)
 
 
+def _need_symbolic_context(symbolic_fn):
+    # Check if the first argument to symbolic_fn is annotated as type `torch.onnx.SymbolicContext`
+    full_arg_spec = inspect.getfullargspec(symbolic_fn)
```

Comment:
Up to you, but maybe slightly more compact and just as readable way of getting the same thing:

```python
params = inspect.signature(symbolic_fn).parameters.values()
return params and issubclass(params[0].annotation, SymbolicContext)
```

It's nice that this API doesn't require us to do the mapping between args and annotations.